### ISSUE TITLE
Picker: accessibility violation

### ISSUE THREAD

**Issue by carolframpton** [adobe/react-spectrum#1951]
# 🐛 Bug Report 

The Picker, for example, https://react-spectrum.adobe.com/react-spectrum/Picker.html#example, has a div with aria-hidden=true which contains an input.

## 🤔 Expected Behavior

No accessibility violations

## 😯 Current Behavior

Our storybook which contains a Picker 

Ensures aria-hidden elements do not contain focusable elements

ARIA hidden element must not contain focusable elements

More info...
div[aria-hidden="true"]

Serious
Focusable content should be disabled or be removed from the DOM

## 🔦 Context

We're aiming for no accessibility errors in our Storybook.

## 💻 Code Sample

From https://react-spectrum.adobe.com/react-spectrum/Picker.html#example
```
<div aria-hidden="true" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); clip-path: inset(50%); height: 1px; margin: 0px -1px -1px 0px; overflow: hidden; padding: 0px; position: absolute; width: 1px; white-space: nowrap;">
<input type="text" tabindex="0" style="font-size: 16px;">
<label>Choose frequency
<select size="3" tabindex="-1">
<option></option>
<option value="rarely">Rarely</option>
<option value="sometimes">Sometimes</option>
<option value="always">Always</option>
</select>
</label>
</div>
```

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3.4.0
| Browser          |
| Operating System |

## 🧢 Your Company/Team

Adobe Admin Console

**Comment by devongovett** [comment]
This is a false positive. We use a hidden native select element to handle form autofill. Focus gets forwarded to the button element. Unfortunately, there's not much we can do to suppress the warning. I'll forward you an internal slack thread from someone else on your team where we already answered this as well.
